### DISCUSSION TITLE
Determining dimensions of output layers of an ONNX model

### DISCUSSION THREAD

**Discussion by Shassk** [ultralytics/yolov5#7274] [Q&A]
I have already pre-trained YOLOv5 model in ONNX format. How can I find out what are the number and dimensions of it's output layers? Plaease answer as if I knew only that a neural network recieves an input array and gives another array in return.

**Comment by glenn-jocher** [comment]
Use Netron to inspect inputs and outputs.

**Reply by Shassk** [reply]
Thanks, I've installed it and I see 4 endpoints:

- 1x25200x6 (which I honestly don't understand)
- 1x3x20x20x6
- 1x3x40x40x6
- 1x3x80x80x6

On previous model I've had there were 2 endpoints 1x90x13x13 and 1x90x26x26 - and the output was parsed as 2 arrays (13x13 and 26x26), but here I can't really get what's going on

Any ideas here? Or it'll be hard to answer w/o the model itself?
